### PR TITLE
Add Debian packaging

### PR DESCRIPTION
**Pull request by dberliner**
This should have been pushed weeks ago. We say we maintain a Debian package in the docs but don't actually do so... This is the Debian branch that produces the package to work with DKMS.

### CODE REVIEW

File: host/debian/files [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hss-dkms_0.1.0-1_all.deb admin optional
```
Comment (drbild) [inline host/debian/files]: What is the file (named "file"!) for?

**Review by drbild — CHANGES_REQUESTED** [review]
One question about the purpose of the `debian/files` file.

File: host/debian/files (hunk 1 above)
Comment (dberliner) [reply host/debian/files]: Nothing, it would seem (just a remnant from an example I copied). I removed it without consequence so I'll push that remove in.

**Review by drbild — APPROVED** [review]
LGTM.

**Pull request merged**